File: 2744/2744.py
# **************************************************************************** #
#                                                                              #
#                                                         :::      ::::::::    #
#    2744.py                                            :+:      :+:    :+:    #
#                                                     +:+ +:+         +:+      #
#    By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+         #
#                                                 +#+#+#+#+#+   +#+            #
#    Created: 2024/03/03 13:47:36 by jihoolee          #+#    #+#              #
#    Updated: 2024/03/03 13:51:58 by jihoolee         ###   ########.fr        #
#                                                                              #
# **************************************************************************** #

def swap_case(input_: str) -> str:
    swapped_word: str = ''

    for char in input_:
        if char.islower():
            swapped_word += char.upper()
        elif char.isupper():
            swapped_word += char.lower()
        else:
            swapped_word += char
    return swapped_word


def main() -> None:
    print(swap_case(input()))


if __name__ == '__main__':
    main()

File: 2744/2744.cpp
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   2744.cpp                                           :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2024/03/03 13:38:00 by jihoolee          #+#    #+#             */
/*   Updated: 2024/03/03 13:47:12 by jihoolee         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

#include <iostream>
#include <string>
#include <cctype>

std::string swap_case(const std::string& word) {
  std::string swapped_word;

  for (size_t idx = 0; idx < word.size(); ++idx) {
    if (islower(word[idx]))
      swapped_word += toupper(word[idx]);
    else if (isupper(word[idx]))
      swapped_word += tolower(word[idx]);
    else
      swapped_word += word[idx];
  }
  return swapped_word;
}

int main(void) {
  std::string word;

  std::cin >> word;
  std::cout << swap_case(word);
  return 0;
}
